demo/src/main/java/com/wishes/demo/repository/converter/access_status/AccessStatusAttributeConverter.java

```diff
@@ -0,0 +1,28 @@
+package com.wishes.demo.repository.converter.access_status;
+
+import javax.persistence.AttributeConverter;
+import javax.persistence.Converter;
+
+
+@Converter(autoApply = true)
+public class AccessStatusAttributeConverter implements AttributeConverter<Boolean,Integer> {
```

Comment:
No need to manually add converter for _boolean_, Hibernate will convert it, remove this one